src/asynqp/__init__.py

```diff
@@ -19,8 +19,12 @@
 def connect(host='localhost',
             port=5672,
             username='guest', password='guest',
-            virtual_host='/', *,
-            loop=None, sock=None, **kwargs):
+            virtual_host='/',
+            *,
+            loop=None,
+            sock=None,
+            nodelay=True,
```

Comment:
I think this is very bad idea since it would be https://en.wikipedia.org/wiki/Overengineering.